docs/styles/spec/file-input-spec.less

```diff
@@ -1,5 +1,5 @@
 @import (reference) 'components/button/index';
-@import (reference) 'components/file/index';
+//@import (reference) 'components/file/index';
```

Comment:
Put back the LESS file so that we don't break this spec. We can update the spec to SCSS in a later PR.